src/components/splashScreen/TitleBadge.tsx

```diff
@@ -7,17 +7,31 @@ type TitleBadgeProps = {
 };
 
 export default function TitleBadge(props: TitleBadgeProps) {
-  const { isPaniniOrdered } = useOutletContext() as LayoutContext;
+  const { userStep, setOrderData } = useOutletContext() as LayoutContext;
```

Comment:
**nitpick:** You don't need user step because you can access current route from `useRouter` (as I recall, or maybe `usePathname?`)